docs/_pages/contribute/06-outreach.md

```diff
@@ -19,7 +19,7 @@ The best we can do is reason with users of why they should value their privacy a
 
 Social media is a great opportunity for word-of-mouth advertising of RoboSats. If you had a great experience using RoboSats or have interesting insight to share about the project, please do say something! Shout it from the robo-heavens!
 
-For example, you could create a tweet and tag the [RoboSats Twitter](https://twitter.com/RoboSats) account so that we can retweet it! Note that not all tweets are guaranteed retweets. Similarly, you could post on Nostr and tag RoboSats Nostr (npub1p2psats79rypr8lpnl9t5qdekfp700x660qsgw284xvq4s09lqrqqk3m82) account so that it can be shared with others.
+For example, you could post on Nostr and tag the RoboSats Nostr account (npub1p2psats79rypr8lpnl9t5qdekfp700x660qsgw284xvq4s09lqrqqk3m82) so that it can be shared with others!
```

Comment:
Sounds again like we are fully dropping Twitter. Not against it, I understand we can not rely on it, but it is still the main outreach place.